understanding/22/accessible-authentication-enhanced.html

```diff
@@ -97,7 +98,7 @@ <h2>Techniques for Accessible Authentication (Enhanced)</h2>
       
       
       <section id="sufficient">
-         <h3>Sufficient Techniques for Accessible Authentication</h3>
+         <h3>Sufficient Techniques for Accessible Authentication (Enhanced)</h3>
```

Comment:
I don't see the SC title being put into any of the sub-headings in other SCs. It would normally just be "Sufficient Techniques"
```suggestion
         <h3>Sufficient Techniques</h3>
```